core/schema_utils.py

```diff
@@ -663,3 +663,17 @@ def is_valid_username_string(obj: str) -> bool:
             return True
         except utils.ValidationError:
             return False
+
+    @staticmethod
+    def is_at_most_char(obj: str, max_value: int) -> bool:
```

Comment:
We already have `has_length_at_most`

Reply:
Sorry, I missed it. I have deleted the new one and added `has_length_at_most` in the validator.